meredithisme/psuedocode.txt

```diff
@@ -0,0 +1,15 @@
+
+
+GET ALL ICECREAM DATA “/icecream”
+
+GET ICREAM FLAVORS BY ID “/icecream/flavors/:id"
+
+GET ALL ICECREAM FLAVORS “/icecream/flavors”
+
+GET ICECREAM TEXTURES “/icecream/textures”
+
+GET ALL TEXTURES BY FLAVOR “/flavors/:id?=/textures”
```

Comment:
maybe you could specify a specific flavor and texture here
`/flavors/:id?texture=`
Which would be something like `localhost:3000/flavors/31?texture=creamy`